staging/copy.sh

```diff
@@ -102,10 +102,9 @@ find "${MAIN_REPO}/pkg/version" -maxdepth 1 -type f | xargs -I{} cp {} "${CLIENT
 # need to copy clientsets, though later we should copy APIs and later generate clientsets
 mkcp "pkg/client/clientset_generated/${CLIENTSET}" "pkg/client/clientset_generated"
 mkcp "/pkg/client/record" "/pkg/client"
-
 mkcp "/pkg/client/unversioned/portforward" "/pkg/client/unversioned"
-
 mkcp "/pkg/util/workqueue" "pkg/util"
+mkcp "/pkg/api/testing" "pkg/api"
```

Comment:
We could use gsed instead in fact, forcing Mac users into installing it (which is acceptible IMO).